playground/package.json

```diff
@@ -2,74 +2,36 @@
   "name": "merchant-center-starter-app",
   "version": "1.0.0",
   "description": "A starter example for building a Merchant Center application",
-  "main": "index.js",
   "scripts": {
-    "commitmsg": "commitlint -e $GIT_PARAMS",
-    "precommit": "lint-staged",
     "build": "mc-scripts build",
     "start": "mc-scripts start",
     "start:prod": "mc-http-server --config $(pwd)/env.prod.json --csp $(pwd)/csp.json --use-local-assets",
     "now-build": "NODE_ENV=production npm run build",
-    "now-start": "NODE_ENV=production npm run start:prod",
-    "lint": "jest --projects jest.eslint.config.js",
-    "format": "npm run format:js && npm run format:md",
-    "format:js": "prettier --write 'src/**/*.js'",
-    "format:md": "prettier --parser markdown --write '*.md,src/**/*.md'",
-    "test": "jest --config jest.test.config.js",
-    "test:watch": "jest --config jest.test.config.js --watch",
-    "ci": "jest --projects jest.*.config.js"
+    "now-start": "NODE_ENV=production npm run start:prod"
```

Comment:
In theory, we can install `now` to automatically build/deploy the app. It won't be useful for branch deployments though and for checking the UI because of CORS restrictions if you open the browser at a domain that is not whitelisted.

For now I'll keep it without but we might consider looking into that workflow in the future.